lib/web_console/evaluator.rb

```diff
@@ -14,8 +14,8 @@ def initialize(binding = TOPLEVEL_BINDING)
       @binding = binding
     end
 
-    def eval(input)
-      "=> #{@binding.eval(input).inspect}\n"
+    def eval(input, rawdata = nil)
+      rawdata ? @binding.eval(input) : "=> #{@binding.eval(input).inspect}\n"
```

Comment:
Instead of introducing a condition here, can we save the current selected binding in the session object? That way, we can evaluate code in an evaluator object and inspect the variables with the binding itself.